resources/views/components/newsletter.blade.php

```diff
@@ -8,12 +8,15 @@
         {{ $attributes->class('flex bg-white p-7 cursor-pointer relative rounded border') }}
     >
         <x-rapidez-ct::input.checkbox
-            class="!items-start"
+            class="w-full !items-start"
             id="{{ $id }}"
         >
             <x-slot:slot class="ml-2 flex flex-col gap-1">
                 <span class="text-sm font-medium text-ct-primary">@lang('Yes, I want to subscribe to the newsletter')</span>
                 <span class="text-xs font-normal text-ct-inactive">@lang('You will receive this newsletter approximately 2x a year')</span>
+                <div class="-translate-y-1/2 absolute right-0 top-1/2 max-sm:hidden">
+                    <x-rapidez-ct::newsletter-visual />
+                </div>
```

Comment:
hmm, isnt this "checkbox component" used more often? like the delivery method, payment method, and also for the orders within the customer centre, maybe it's nice to create a component for this instead of specific styling like this. What do you think?